tests/protocol/test_wire.py

```diff
@@ -83,15 +84,20 @@ def test_getters_with_stale_goal_state(self, *args):
         test_data = WireProtocolData(DATA_FILE)
         test_data.emulate_stale_goal_state = True
 
-        self._test_getters(test_data, *args)
+        try:
+            self._test_getters(test_data, *args)
+            # Expect this to throw a ProtocolError because of stale goal state
+            self.assertTrue(False)
+        except ProtocolError:
+            pass
         # Ensure HostPlugin was invoked
-        self.assertEqual(1, test_data.call_counts["/versions"])
-        self.assertEqual(2, test_data.call_counts["extensionArtifact"])
+        #self.assertEqual(1, test_data.call_counts["/versions"])
```

Comment:
Should these be commented out?

Reply:
They no longer get triggered because the intended behavior that this tests for has changed. Removed.